pkg/distributor/ha_tracker_test.go

```diff
@@ -271,7 +291,7 @@ func TestHaTrackerWithMemberList(t *testing.T) {
 	assert.Error(t, err)
 
 	// Wait more than the overwrite timeout.
-	now = now.Add(1100 * time.Millisecond)
+	now = now.Add(time.Millisecond * failoverTimeoutPlus100ms)
```

Comment:
this is multiplying milliseconds by milliseconds, was that the intention?

Reply:
No, it was me, forgetting to remove it 😭 . Updated it here [2ceeadc](https://github.com/grafana/mimir/pull/10020/commits/2ceeadc8c9dcb16ebd9540b074022255be21c72f)